includes/controls/animation.php

```diff
@@ -160,4 +160,17 @@ public function content_template() {
 		<# } #>
 		<?php
 	}
+
+	public static function get_conditional_assets( $setting ) {
+		if ( ! $setting ) {
+			return [];
+		}
+
+		return [
+			[
```

Comment:
I think a better structure for this array would be something like this:

```
[
    'styles' => [ 'e-animations', 'sticky', etc. ],
]

Reply:
Fixed.